server/portal/apps/workspace/api/views.py

```diff
@@ -125,27 +127,16 @@ def get(self, request, *args, **kwargs):
 
 @method_decorator(login_required, name='dispatch')
 class JobsView(BaseApiView):
-    def get(self, request, *args, **kwargs):
+    def get(self, request, operation=None):
         tapis = request.user.tapis_oauth.client
-        job_uuid = request.GET.get('job_uuid')
 
-        # get specific job info
-        if job_uuid:
+        if operation == 'select': 
+            job_uuid = request.GET.get('job_uuid')
             data = tapis.jobs.getJob(jobUuid=job_uuid)
-
-        # list jobs
-        else:
-            limit = int(request.GET.get('limit', 10))
-            offset = int(request.GET.get('offset', 0))
-            portal_name = settings.PORTAL_NAMESPACE
-
-            data = tapis.jobs.getJobSearchList(
-                limit=limit,
-                startAfter=offset,
-                orderBy='lastUpdated(desc),name(asc)',
-                _tapis_query_parameters={'tags.contains': portal_name},
-                select='allAttributes'
-            )
+        elif operation == 'listing':
+            data = self.listing(tapis, request)
+        elif operation == 'search':
+            data = self.search(tapis, request)
```

Comment:
Another way to do this a bit more dynamically:

```suggestion
        op = getattr(self, operation)
        data = op(tapis, request)
```